scripts/sizeSnapshot/worker.js

```diff
@@ -0,0 +1,62 @@
+const { promisify } = require('util');
+const webpackCallbackBased = require('webpack');
+const { createWebpackConfig } = require('./webpack.config');
+
+const webpack = promisify(webpackCallbackBased);
+
+async function getSizes({ entry, webpackEnvironment, index, total }) {
+  const sizes = [];
+
+  const configuration = createWebpackConfig(entry, webpackEnvironment);
+
+  // eslint-disable-next-line no-console -- process monitoring
+  console.log(`Compiling ${index}/${total}: "${Object.keys(configuration.entry)}"`);
```

Comment:
Minor thing - it makes more sense to use human-readable 1-based indices at this point.

```suggestion
  console.log(`Compiling ${index + 1}/${total}: "${Object.keys(configuration.entry)}"`);
```